test/runtests.jl

```diff
@@ -90,12 +90,16 @@ else
     @test start(t) == nothing
     @test length(read(t, UInt8, 6)) == 12
     @test stop(t) == nothing
-    @test NIDAQ.CfgSampClkTiming(t.th, convert(Ref{UInt8},b""), 100.0, NIDAQ.Val_Rising,
+    rslt = Ref(UInt32(0))
+    NIDAQ.DAQmxGetBufInputOnbrdBufSize(t.th, rslt)
+    if rslt.x != 0 #If the device supports buffered digital input
+        @test NIDAQ.CfgSampClkTiming(t.th, convert(Ref{UInt8},b""), 100.0, NIDAQ.Val_Rising,
               NIDAQ.Val_FiniteSamps, UInt64(10)) == 0
-    @test start(t) == nothing
-    @test length(read(t, UInt32)) == 20
-    @test stop(t) == nothing
-    @test clear(t) == nothing
+	@test start(t) == nothing
+	@test length(read(t, UInt32)) == 20
+	@test stop(t) == nothing
+	@test clear(t) == nothing
+    end
```

Comment:
might be nice to add an `elseif` here stating that device does not support clocked digital input.